### ISSUE TITLE
Docker Breaks During Builds

### ISSUE THREAD

**Issue by egyptiankarim** [dhs-ncats/pshtt#142]
ae5f2e2f15f61704849955f3c8744cf8fc5e0c3d breaks the Docker build:

```
› docker build -t pshtt/cli .
Sending build context to Docker daemon  1.364MB
Step 1/4 : FROM python:3.5-onbuild
# Executing 3 build triggers...
Step 1/1 : COPY requirements.txt /usr/src/app/
Step 1/1 : RUN pip install --no-cache-dir -r requirements.txt
 ---> Running in 2e58c6400f57
Directory '.' is not installable. File 'setup.py' not found.
The command '/bin/sh -c pip install --no-cache-dir -r requirements.txt' returned a non-zero code: 1
```

The operating assumption for the python:3.5-onbuild is that requirements.txt will contain all dependencies, so the build triggers see this and die. I think we fix this with a change in Docker image tag and a few tweaks to the Dockerfile (#141).

**Comment by jsf9k** [comment]
I can fix this tonight. Good catch!

**Comment by jsf9k** [comment]
Fixed via pull request #141.
